with-emotion/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "with-emotion",
+  "displayName": "With emotion",
+  "version": "0.0.0",
+  "description": "A basic Plasmo extension.",
+  "author": "olhapi",
+  "packageManager": "yarn@1.23.0-20220130.1630",
+  "scripts": {
+    "dev": "plasmo dev",
+    "build": "plasmo build"
+  },
+  "dependencies": {
+    "@emotion/cache": "^11.9.3",
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
+    "plasmo": "0.49.0",
```

Comment:
Please follow this guide for new example: https://github.com/PlasmoHQ/plasmo/blob/main/.github/CONTRIBUTING.md#adding-examples

We want to ensure the same pm, and workspace version as well as some other setup are used for example (since it's meant to be a submodule of the larger repo)

Also please use fixed version instead.